.env.example

```
@@ -160,6 +160,14 @@ LOGS_DIR=
 # Used for AI-powered features and model gateway
 OPENROUTER_API_KEY=
 
+# -----------------------------------------------------------------------------
+# Stripe Payments Configuration (Optional)
+# -----------------------------------------------------------------------------
+# Developer-owned Stripe secret keys. Test key is used for implementation and
+# validation. Live key is used only for explicit go-live flows.
+STRIPE_TEST_SECRET_KEY=
+STRIPE_LIVE_SECRET_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The STRIPE_LIVE_SECRET_KEY key should go before the STRIPE_TEST_SECRET_KEY key

(UnorderedKey)